### PR TITLE
Leaderboard: Hit API

### PR DESCRIPTION
**Pull request by fed1k**
# ✔ Followed followings:
✔ Create game with unique id result
✔ Send data to that game
✔ Get data back when 'Reset' button clicked or window reloaded
✔ Use async and await

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+    const raha =  await response.json();
+    for(let i of raha.result){
+        const p = document.createElement('p');
+        p.textContent = `${i.user}: ${i.score}`;
+        listBoard.appendChild(p);
+        
+    }
+}
+// let count = 0;
+reset.addEventListener('click', ()=>{
```
Comment (uche-inyama) [inline src/index.js]: Kindly remove the code commented out, on line 27.

File: src/scoreList.js [hunk 2]
```diff
@@ -1,14 +1,37 @@
...
+      body: JSON.stringify({
+        user: array[0].name,
+        score: array[0].score
+      }),
+      headers: { 'Content-type': 'application/json; charset=UTF-8'}
+    }
+  )
+  const haha = await response.json();
+  console.log(haha);
+}
```
Comment (uche-inyama) [inline src/scoreList.js]: Kindly return a variable from the function and remove console.log.

File: src/scoreList.js [hunk 3]
```diff
@@ -1,14 +1,37 @@
...
+    const list = new List(nameClass.value, scoreClass.value);
+    array.push(list);
+    nameClass.value = null;
+    scoreClass.value = null;
+    func();
+  }
+};
+
+const func = async()=>{
+  const response = await fetch(
```
Comment (uche-inyama) [inline src/scoreList.js]: It would be nice, if you give this function a descriptive name.
### :slightly_smiling_face:

**Review by uche-inyama — CHANGES_REQUESTED** [review]
Hi @fed1k :slightly_smiling_face:  

Good job so far!

There are few updates you need to make.

## Required Changes ♻️

_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# INVALID Code Review Request :stop_sign: 

Hi @fed1k , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that does not run linter checks for lighthouse and HTML because your linters.yml file is not complete.

- [ ]  Kindly, add the missing linters in the linters.yml file;
[Correct linters link](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml);

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

**Comment by fed1k** [comment]
@nicupop729 Hi. One code reviewer told me that I do not need Lighthouse and html linters anymore since this is JavaScript module. That's why I removed them and so far all code reviewers did not make this problem but if you say to include them that is okay I will do that. Thanks for the code review

**Review by nicupop729 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @fed1k, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

We should respect the linters' guidance for all projects.
This applies to all projects across the entire course.
Even if is a project based on JavaScript, we make use of HTML too, which means we should follow good practices still. :+1: :1st_place_medal: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fed1k** [comment]
@nicupop729 Okay I will keep that all the time then. Thank you for the code review again.😊

**Pull request merged**